packages/quantic/force-app/main/default/lwc/quanticNotifications/__tests__/quanticNotifications.test.js

```diff
@@ -224,4 +247,86 @@ describe('c-quantic-notifications', () => {
       });
     });
   });
+
+  describe('when clicking on a notification close button', () => {
+    it('should properly dismiss that notification', async () => {
+      const element = createTestComponent();
+      mockSearchStatusHasResults();
```

Comment:
Why do you need this here?

Reply:
because `this.notifications` is set in `handleSearchStatusChange` which is called when the searchStatus changes:

`this.unsubscribeSearchStatus = this.searchStatus.subscribe(() => this.handleSearchStatusChange());` 

So if I want to trigger a "fake search" which sets the notification, I need to mock a searchStatus change

Reply:
I know but why do you wanna trigger a fake search when the test is about closing a notification?

Reply:
following discussions, we agreed on a solution, the search status is needed for 2 cases: when you dismiss notif and re-do the same query, it should make the same notifs re-appear and also because of a weird glitch when dismissing notifs and triggering another query where the 'old notifs' would appear for a second before the state was updated.